src/ReactiveUI.Tests/ReactiveCollectionTest.cs

```diff
@@ -481,6 +481,64 @@ public void GetAResetWhenWeAddALotOfItems()
             Assert.Equal(1, reset.Count);
         }
 
+        [Fact]
+        public void ActOnEveryObjectShouldHandleAddRemoveAndClear()
```

Comment:
I think from a clarity point of view, this test is doing too much, I think each separate case of ActOnEvery… should be a separate test.